alphagov/paas-cf
# [#131975557] Configure log forwarding with syslog-release

## What

We're removing the old deployment of metron and consul agents, by reverting the previous commits. It will be replaced by our new approach.

We are adding a syslog_forwarder and syslog_cleanup addons to the base runtime config.

The syslog_forwarder addon configures syslog-release to forward syslog messages to the logsearch relp ingestors.

The syslog_cleanup addon configures os-config/cleanup_syslog_forwarding to remove the configuration leftover from the metron_agent.  See alphagov/paas-os-conf-release#3 for more information on this.
## How to review

This is dependant on [paas-os-conf-release pull request](https://github.com/alphagov/paas-os-conf-release/pull/3). Once the other PR is merged, we can remove the TMP commit from this PR and refer to the new release in this one.
- Deploy the pipelines
- Run the `create-bosh-cloudfoundry` pipeline
- See, if correct VMs have the syslog forwarder enabled
- See that VMs no longer have the old `/etc/rsyslog.d/00-syslog_forwarder.conf` present
## Who can review

Not @richardc or @paroxp.
